integration/experiment/common_args.py

```diff
@@ -125,7 +125,7 @@ def add_cool_off_time(parser):
 
 def add_agent_list(parser):
     parser.add_argument('--agent-list', dest='agent_list',
-                        action='store', type=str, default=None,
+                        action='store', type=str, default='power_governor',
```

Comment:
I'm not sure we want this in dev.

Reply:
It looks like ```add_agent_list()``` is only used in power_sweep.py, but power_sweep.py already has an override for the defaults for agent_list.

I'm confused on what bug this was a workaround for?